### PR TITLE
Sound level loud sound

### PR DESCRIPTION
**Pull request by kattni**
Added `sound_level` and `loud_sound` input. Accounted for changes in `PDMIn` in newer versions.

Updated `play_file` to work with 3.0 as well as 2.x.

### CODE REVIEW

File: adafruit_circuitplayground/express.py [hunk 1]
```diff
@@ -99,6 +100,16 @@ def __init__(self):
...
         self._sine_wave = None
 
+        # Define sound sensor/mic:
+        try:
+            self._mic = audiobusio.PDMIn(board.MICROPHONE_CLOCK, board.MICROPHONE_DATA,
+                                         sample_rate=16000, bit_depth=16)
+        except TypeError:
+            self._mic = audiobusio.PDMIn(board.MICROPHONE_CLOCK, board.MICROPHONE_DATA,
+                                         frequency=16000, bit_depth=16)
+        self._samples = array.array('H', [0] * 160)
```
Comment (tannewt) [inline adafruit_circuitplayground/express.py]: 320 bytes is a lot to hang on to. Are you sure its worth always using it?

Comment (kattni) [reply adafruit_circuitplayground/express.py]: I moved the array creation to inside the `sound_level` property.

**Comment by kattni** [comment]
The updates to work with 3.0 should definitely be added to this lib regardless.

However, there are currently some issues with how `sound_level` works with `start_tone`/`stop_tone`. Attempting to resolve these issues. 

I may remove the `sound_level` code and do a separate PR for the 3.0 compatibility updates.

**Comment by kattni** [comment]
Closing this PR. There are too many issues between memory and `audioio` functionality to add `sound_level` and `loud_sound`. It seems possible they would work without `start_tone` and `stop_tone`, but as that functionality already existed, it takes priority. Will revisit once `audioio` issues are resolved in 3.0.

Will be separating 3.0 functionality updates in this PR out into a separate PR and resubmitting.

**Pull request closed without merging**